tutorial/admin/04_setup_default_frequency_map.sh

```diff
@@ -49,6 +49,6 @@ echo "{\"GEOPM_AGENT\": \"energy_efficient\", \"GEOPM_POLICY\": \"$POLICY_FILE_P
 #   > srun geopmread MSR::PERF_CTL:FREQ board 0
 #   3700000000
 #
-# GEOPM jobs use EE agent with above policy; check report
+# GEOPM jobs use frequency map agent with above policy; check report
 #   > geopmlaunch srun -N1 -n1 --geopm-report=plugin_test.report -- geopmbench ~/short.conf > geopm_stdout 2>&1 && grep Policy plugin_test.report
-#   Policy: {"FREQ_MIN": 1200000000, "FREQ_MAX": 1700000000}
```

Comment:
why did 1.7 GHz -> 1.8 GHz?

Reply:
No special reason. This example was inspired by example 03. It obtains the CPU frequency sticker and subtracts 300MHz from it. It shows 1.8GHz as a hypothetical result. To make the examples different enough, I'll make this one simply set a fixed frequency of 1.7GHz (as the max that was being used before).